src/Components/CacheAdapter.php

```diff
@@ -133,4 +142,16 @@ private function getPathOfFilesAdapter(PhpFilesAdapter $adapter): string
 
         return $getter($adapter);
     }
+
+
+    private function getServerFreeMemory(): int
```

Comment:
BAHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH

NEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Server != REDIS

Reply:
you're right :-/